### ISSUE TITLE
Electron?

### ISSUE THREAD

**Issue by knipferrc** [lukeed/pwa#32]
Looking to build an svelte electron app without worrying much about config, thought this might be a good starting point, anyone tried it with electron yet or is it possible? Had it somewhat working but ran into issues with routing in the prod build.

**Comment by lukeed** [comment]
Yup! I'm actually of the same mindset right now. Building an Electron app for work and think it can totally be a preset.

Have not opened a ticket because I'm still uncertain if this can be done in a reliable way.
